### PR TITLE
chore(example): Update sample app

### PR DESCRIPTION
**Pull request by hiteshjain29**
### Changelog:
- Rebranded Sample App 

Simulator Screen Shot - iPhone 11 - 2020-07-24 at 17 36 19

Simulator Screen Shot - iPhone 11 - 2020-07-24 at 17 36 22

### CODE REVIEW

File: UsabillaSampleApp/.gitignore [hunk 1]
```diff
@@ -11,3 +11,5 @@ plugins/
 
 # res
 resources/signing
+resources/android/
+resources/ios/
```
Comment (kioli) [inline UsabillaSampleApp/.gitignore:15]: EOF needed

File: UsabillaSampleApp/README.md [hunk 2]
```diff
@@ -26,4 +35,124 @@ $ cordova-res
...
-Otherwise, if `config.xml` exists, `cordova-res` will look for platforms (e.g. `<platform name="ios">`) and generate resources only for the configured platforms.
\ No newline at end of file
+Otherwise, if `config.xml` exists, `cordova-res` will look for platforms (e.g. `<platform name="ios">`) and generate resources only for the configured platforms.
+
+### Add Platform
+
+- To add native platforms like android & ios , just run:
+
+```bash
+$ cordova platform add android ios
```
Comment (kioli) [inline UsabillaSampleApp/README.md:45]: can you actually have them both in the same command or should they be two separate commands?

File: UsabillaSampleApp/README.md [hunk 3]
```diff
@@ -26,4 +35,124 @@ $ cordova-res
...
+```
+
+### Add Usabilla plugin
+
+- To add usabilla into cordova application , just run:
```
Comment (kioli) [inline UsabillaSampleApp/README.md:50]: To add the Usabilla SDK into your Cordova application, just run:

File: UsabillaSampleApp/README.md [hunk 4]
```diff
@@ -26,4 +35,124 @@ $ cordova-res
...
+/// Usabilla Configuration
+const appId = 'YOUR_APP_ID_HERE';
+const formId = 'YOUR_FORM_ID_HERE';
+let event = 'YOUR_EVENT_TAG_HERE';
+const customVariable = {'YOUR_KEY_HERE': 'YOUR_VALUE_HERE'};
+```
+
+### Functions
+
+- Initialize the sdk and sets custom variables for targeting Campaigns.
```
Comment (kioli) [inline UsabillaSampleApp/README.md:72]: `Initialize the SDK and set custom variables for targeting campaigns`

File: UsabillaSampleApp/README.md [hunk 5]
```diff
@@ -26,4 +35,124 @@ $ cordova-res
...
+```
+initialize → String appId , Object customVariable
+```
+``` js
+    const customVariable = {'YOUR_KEY_HERE': 'YOUR_VALUE_HERE'};
+    Usabilla.initialize(function() {}, function (err) {}, appId, customVariable);
+```
+- Shows the Passive Form & Returns a JSON Object, contains result - rating, pageindex, sent flag.
```
Comment (kioli) [inline UsabillaSampleApp/README.md:80]: `Show the passive form and return a JSON Object containing the results - rating, pageIndex and sent flag`

File: UsabillaSampleApp/README.md [hunk 6]
```diff
@@ -26,4 +35,124 @@ $ cordova-res
...
+```
+loadFeedbackForm → String formId
+```
+``` js
+    Usabilla.loadFeedbackForm(function() {}, function (err) {}, formId);
+```
+- Load the Passive Form with current screen captured & Returns a JSON Object, contains result - rating, pageindex, sent flag.
```
Comment (kioli) [inline UsabillaSampleApp/README.md:87]: `Load the passive form with the current screen captured and return a JSON Object  containing the results - rating, pageIndex and sent flag`

File: UsabillaSampleApp/README.md [hunk 7]
```diff
@@ -26,4 +35,124 @@ $ cordova-res
...
+```
+loadFeedbackFormWithCurrentViewScreenshot → String formId
+```
+``` js
+    Usabilla.loadFeedbackFormWithCurrentViewScreenshot(function() {}, function (err) {}, formId);
+```
+- Load the Campaign & Returns a Map, contains result - rating, pageindex, sent flag.
```
Comment (kioli) [inline UsabillaSampleApp/README.md:94]: `Load the campaign and return a map containing the results - rating, pageIndex and sent flag`

File: UsabillaSampleApp/README.md [hunk 8]
```diff
@@ -26,4 +35,124 @@ $ cordova-res
...
+```
+sendEvent → String event
+```
+``` js
+    Usabilla.sendEvent(function() {}, function (err) {}, event);
+```
+- Reset Campaign, so can be triggered from fresh count.
```
Comment (kioli) [inline UsabillaSampleApp/README.md:101]: `Reset campaign data, so they can be triggered again from a fresh count`

File: UsabillaSampleApp/README.md [hunk 9]
```diff
@@ -26,4 +35,124 @@ $ cordova-res
...
+```
+resetCampaignData()
+```
+``` js
+    Usabilla.resetCampaignData(function() {}, function (err) {});
+```
+- Manually Dismiss the Forms / Campaign.
```
Comment (kioli) [inline UsabillaSampleApp/README.md:108]: `Dismiss the currently visible form`

File: UsabillaSampleApp/README.md [hunk 10]
```diff
@@ -26,4 +35,124 @@ $ cordova-res
...
+```
+dismiss()
+```
+``` js
+    Usabilla.dismiss(function() {}, function (err) {});
+```
+- Sets data masking with default character / passed single character and based on masks rule.
```
Comment (kioli) [inline UsabillaSampleApp/README.md:115]: `Set data masking based on passed/default rules using a passed/default character for obfuscation`

File: UsabillaSampleApp/README.md [hunk 11]
```diff
@@ -26,4 +35,124 @@ $ cordova-res
...
+```
+setDataMasking → Array masks, String character
+```
+``` js
+    const masks = 'YOUR_Masking_Regex';
+    const maskCharacter = 'YOUR_Masking_Character';
+    Usabilla.setDataMasking(function() {}, function (err) {}, masks, maskCharacter);
+```
+- Returns masks array to check what is the defined rule.
```
Comment (kioli) [inline UsabillaSampleApp/README.md:124]: `Return the array of default masking rules`

File: UsabillaSampleApp/README.md [hunk 12]
```diff
@@ -26,4 +35,124 @@ $ cordova-res
...
+```
+getDefaultDataMasks()
+```
+``` js
+    Usabilla.getDefaultDataMasks(function() {}, function (err) {});
+```
+- Store forms in a cache (file storage), which can be used in a offline mode.
```
Comment (kioli) [inline UsabillaSampleApp/README.md:131]: `Store form(s) in memory to be used in offline mode`

File: UsabillaSampleApp/README.md [hunk 13]
```diff
@@ -26,4 +35,124 @@ $ cordova-res
...
+    Usabilla.preloadFeedbackForms(function() {}, function (err) {}, formIds);
+```
+- Remove cached forms stored for offline use.
+```
+removeCachedForms()
+```
+``` js
+    Usabilla.removeCachedForms(function() {}, function (err) {});
+```
+- Remove cached forms stored for offline use.
```
Comment (kioli) [inline UsabillaSampleApp/README.md:145]: Duplicate line

probably we can say `Set debug mode to enable logging on console of Usabilla SDK errors and informations `

File: UsabillaSampleApp/README.md [hunk 14]
```diff
@@ -26,4 +35,124 @@ $ cordova-res
...
+```
+setDebugEnabled → Boolean debugEnabled
+```
+``` js
+    Usabilla.setDebugEnabled(function() {}, function (err) {}, debugEnabled);
+```
+- Sets filename which is used for localization and available only for `iOS`.
```
Comment (kioli) [inline UsabillaSampleApp/README.md:152]: `Set filename used for localization [Available only for iOS]`

File: UsabillaSampleApp/www/css/index.css [hunk 15]
```diff
@@ -16,106 +16,138 @@
...
+      font-weight: 500;
+      letter-spacing: 0;
+      line-height: 19px;
+      border-radius: 2px;
+      background-color: #00A5C9;
+      width: 334px;
+      height: 48px;
+  }
+  
+  .event-group {
```
Comment (kioli) [inline UsabillaSampleApp/www/css/index.css:84]: Does this need to exist is empty?

File: UsabillaSampleApp/www/css/index.css [hunk 16]
```diff
@@ -16,106 +16,138 @@
...
+      /* position: fixed; */
+      margin-top: 50%;
+      -webkit-transform: translateY(-50%);
+      -moz-transform: translateY(-50%);
+      -ms-transform: translateY(-50%);
+      -o-transform: translateY(-50%);
+      transform: translateY(-50%);
+  }
+  
+  .show-form {
```
Comment (kioli) [inline UsabillaSampleApp/www/css/index.css:68]: Does this need to exist is empty?

File: UsabillaSampleApp/www/css/index.css [hunk 17]
```diff
@@ -16,106 +16,138 @@
...
+      font-weight: bold;
+      letter-spacing: 0;
+      margin-left: 30px;
+      margin-top: 30px;
+      line-height: 65px;
+  }
+  
+  .ub-group {
+      text-align:center;
+      /* position: fixed; */
```
Comment (kioli) [inline UsabillaSampleApp/www/css/index.css:59]: Can the comment be removed?

File: UsabillaSampleApp/www/css/index.css [hunk 18]
```diff
@@ -16,106 +16,138 @@
...
+    font-family: 'MiloOT-Medi';
+    src: url('../fonts/MiloOT-Medi.otf');
+  }
 
+ body {
+    background-image: url('../img/background.png');
+    background-size: 100%;
+  }
+  
+  .app {
```
Comment (kioli) [inline UsabillaSampleApp/www/css/index.css:40]: Does this need to exist is empty?

File: UsabillaSampleApp/www/css/index.css [hunk 19]
```diff
@@ -16,106 +16,138 @@
...
+      width: 334px;
+      height: 48px;
+  }
+  
+  .event-group {
+      /* text-align: center; */
+  }
+  
+  .show-event {
+      /* text-align: center; */
```
Comment (kioli) [inline UsabillaSampleApp/www/css/index.css:89]: Can the comment be removed?

File: UsabillaSampleApp/www/css/index.css [hunk 20]
```diff
@@ -16,106 +16,138 @@
...
+      box-sizing: border-box;
+      height: 48px;
+      width: 130px;
+      border: 1px solid #00A5C9;
+      border-radius: 2px;
+      background-color: #FFFFFF;
+  }
+  
+  .reset-event {
+      /* text-align: center; */
```
Comment (kioli) [inline UsabillaSampleApp/www/css/index.css:123]: Can the comment be removed?

File: UsabillaSampleApp/www/index.html [hunk 21]
```diff
@@ -37,24 +37,27 @@
...
         </div>
+        <div class="footer">
+            <img src="img/footer.png" >
+        </div>
         <script type="text/javascript" src="cordova.js"></script>
+        <script type="text/javascript" src="js/config.js"></script>
         <script type="text/javascript" src="js/index.js"></script>
     </body>
-</html>
+</html>
```
Comment (kioli) [inline UsabillaSampleApp/www/index.html:63]: EOF needed

File: UsabillaSampleApp/www/js/index.js [hunk 22]
```diff
@@ -1,4 +1,5 @@
-/*
+
```
Comment (kioli) [inline UsabillaSampleApp/www/js/index.js:1]: Unnecessary empty line

File: UsabillaSampleApp/README.md (hunk 2 above)
Comment (hiteshjain29) [reply UsabillaSampleApp/README.md:45]: It can be in same command. that's not an issue.

Comment (kioli) [reply UsabillaSampleApp/README.md:45]: perfect

File: UsabillaSampleApp/www/css/index.css (hunk 17 above)
Comment (hiteshjain29) [reply UsabillaSampleApp/www/css/index.css:59]: It's kept for future purpose.

File: UsabillaSampleApp/www/css/index.css (hunk 16 above)
Comment (hiteshjain29) [reply UsabillaSampleApp/www/css/index.css:68]: It's kept for future purpose.

File: UsabillaSampleApp/www/css/index.css (hunk 15 above)
Comment (hiteshjain29) [reply UsabillaSampleApp/www/css/index.css:84]: It's kept for future purpose.

File: UsabillaSampleApp/www/css/index.css (hunk 18 above)
Comment (hiteshjain29) [reply UsabillaSampleApp/www/css/index.css:40]: It's kept for future purpose.

File: UsabillaSampleApp/www/css/index.css (hunk 19 above)
Comment (hiteshjain29) [reply UsabillaSampleApp/www/css/index.css:89]: It's kept for future purpose. but can be removed from scope and kept it somewhere with the comments

File: UsabillaSampleApp/www/css/index.css (hunk 20 above)
Comment (hiteshjain29) [reply UsabillaSampleApp/www/css/index.css:123]: It's kept for future purpose.

File: UsabillaSampleApp/www/index.html (hunk 21 above)
Comment (hiteshjain29) [reply UsabillaSampleApp/www/index.html:63]: Done

File: UsabillaSampleApp/www/js/index.js (hunk 22 above)
Comment (hiteshjain29) [reply UsabillaSampleApp/www/js/index.js:1]: Done

File: UsabillaSampleApp/README.md (hunk 14 above)
Comment (hiteshjain29) [reply UsabillaSampleApp/README.md:152]: Done

File: UsabillaSampleApp/README.md (hunk 13 above)
Comment (hiteshjain29) [reply UsabillaSampleApp/README.md:145]: Done

File: UsabillaSampleApp/README.md (hunk 12 above)
Comment (hiteshjain29) [reply UsabillaSampleApp/README.md:131]: Done

File: UsabillaSampleApp/README.md (hunk 11 above)
Comment (hiteshjain29) [reply UsabillaSampleApp/README.md:124]: Done

File: UsabillaSampleApp/README.md (hunk 10 above)
Comment (hiteshjain29) [reply UsabillaSampleApp/README.md:115]: Done

File: UsabillaSampleApp/README.md (hunk 9 above)
Comment (hiteshjain29) [reply UsabillaSampleApp/README.md:108]: Done

File: UsabillaSampleApp/README.md (hunk 8 above)
Comment (hiteshjain29) [reply UsabillaSampleApp/README.md:101]: Done

File: UsabillaSampleApp/README.md (hunk 7 above)
Comment (hiteshjain29) [reply UsabillaSampleApp/README.md:94]: Done

File: UsabillaSampleApp/README.md (hunk 6 above)
Comment (hiteshjain29) [reply UsabillaSampleApp/README.md:87]: Done

File: UsabillaSampleApp/README.md (hunk 5 above)
Comment (hiteshjain29) [reply UsabillaSampleApp/README.md:80]: Done

File: UsabillaSampleApp/README.md (hunk 4 above)
Comment (hiteshjain29) [reply UsabillaSampleApp/README.md:72]: Done

File: UsabillaSampleApp/README.md (hunk 3 above)
Comment (hiteshjain29) [reply UsabillaSampleApp/README.md:50]: Done

File: UsabillaSampleApp/.gitignore (hunk 1 above)
Comment (hiteshjain29) [reply UsabillaSampleApp/.gitignore:15]: Done

**Pull request merged**